__tests__/app.test.js

```diff
@@ -27,10 +27,20 @@ describe("GET /api", () => {
               queries: expect.any(Object),
             });
           }
-          //need to add format body, once I can see how it looks
         }
       });
   });
+  test("GET 200 - Responds with an object describing all the available endpoints on your API", () => {
+    const expectedOutput = JSON.parse(
+      fs.readFileSync("./endpoints.json", "utf8")
+    );
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body: { endpoints } }) => {
+        expect(endpoints).toEqual(expectedOutput);
+      });
+  });
```

Comment:
Not a dealbreaker - but keep in mind JSON is very accessible within javascript, you're allowed to require in the file as a const and access it directly, you don't even have to fs.readFile to do this :)